plasmapy/physics/tests/test_parameters.py

```diff
@@ -705,6 +708,129 @@ def test_Debye_number():
         assert Debye_number(1.1 * u.K, 1.1) == Debye_number(1.1, 1.1 * u.m ** -3)
 
 
+class Test__matrix_sheath_thickness:
+    r"""Test the matrix_sheath_thickness function in parameters.py."""
+
+    V_0 = 80 * u.V
+
+    def test_correct_unit(self):
+        assert matrix_sheath_thickness(self.V_0, n_e).unit.is_equivalent(u.m), \
+            (f"The unit of matrix_sheath_thickness is not meter.")
+
+    def test_correct_value(self):
+
+        test_result = matrix_sheath_thickness(self.V_0, n_e).value
+
+        expected = 1.329823977068723e-05
+
+        test = np.isclose(test_result, expected)
+
+        errStr = (f"The test result value of matrix_sheath_thickness is not correct "
+                  f"and returns {test_result} instead of {expected}.")
+
+        assert test, errStr
```

Comment:
I really think we need to start using http://docs.astropy.org/en/stable/api/astropy.tests.helper.assert_quantity_allclose.html?highlight=assert this function rather than run our custom checks! I'll just keep grumbling in reviews about it, I guess.